gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -427,8 +435,13 @@ def convert_to_lowest_common_time_denom(self, raw_files: list, keep_originals: b
         """
         if len(raw_files) == 0:
             raise ValueError("No files found to convert, exiting script")
-        command_text = ["cdo", "-f", "nc4", "splitsel,1"]
-        self.parallel_subprocess_files(raw_files, command_text, "", keep_originals)
+        command_text = ["cdo", "-f", "nc4c", "splitsel,1"]
+        self.parallel_subprocess_files(
+            input_files=raw_files,
+            command_text=command_text,
+            replacement_suffix=".nc4",
+            keep_originals=keep_originals,
+        )
 
     def ncs_to_nc4s(self, keep_originals: bool = False):
```

Comment:
Maybe make note of the fact that `ncs_to_nc4s` is no longer in use

Reply:
it’s actually still used by some managers that don’t convert via CDO